packages/niftysave/src/migrate/erc721-import.js

```diff
@@ -0,0 +1,35 @@
+import { script } from 'subprogram'
+import * as Migration from '../migrate.js'
+/**
+ * @typedef {{
+ *    id: string
+ *    nextID: string
+ * }} ERC721ImportResult
+ *
+ * @param {Migration.Document<ERC721ImportResult>} document
+ */
+const insert = ({ ts, data: { id, nextID } }) => ({
+  id,
+  next_id: nextID,
+  inserted_at: Migration.fromTimestamp(ts),
+})
+
+/**
+ * @param {Migration.Document<ERC721ImportResult>[]} documents
+ * @returns {Migration.Mutation}
+ */
+export const mutation = (documents) => ({
+  insert_erc721_import: [
+    {
+      objects: documents.map(insert),
```

Comment:
@the-simian should we let it crash or should we ignore conflicts here ?

Reply:
let it crash because we shouldn't ever go back in time/cursor, and log the exception imo